helpertools/fastqc_module.py

```diff
@@ -0,0 +1,22 @@
+def gc_check(sequence, gc_bounds):  # accepts a string - sequence of nucleotides from FASQC-dictionary - and a tuple of too numeric values -  lower and upper bounds for gc-content in provided string
+    gc_count = 0
+    gc_count = sum(True for nucl in list(sequence) if nucl == "G" or nucl == "C")
+    gc = 0
+    gc = gc_count/len(sequence)*100
```

Comment:
```suggestion
    gc = gc_count / len(sequence) * 100
```

операнды разделяем пробелами

-1